packages/environment/src/releases.ts

```diff
@@ -120,13 +128,44 @@ export interface DeploymentNamedTokensPolygon {
   readonly weth: Address;
 }
 
-export type DeploymentNamedTokens<TDeployment extends Deployment> = TDeployment extends Deployment.ETHEREUM
-  ? DeploymentNamedTokensEthereum
-  : TDeployment extends Deployment.POLYGON
-    ? DeploymentNamedTokensPolygon
-    : TDeployment extends Deployment.TESTNET
+export type DeploymentNamedTokens<TDeployment extends Deployment> = TDeployment extends Deployment.ARBITRUM
+  ? DeploymentNamedTokensArbitrum
+  : TDeployment extends Deployment.ETHEREUM
+    ? DeploymentNamedTokensEthereum
+    : TDeployment extends Deployment.POLYGON
       ? DeploymentNamedTokensPolygon
-      : never;
+      : TDeployment extends Deployment.TESTNET
+        ? DeploymentNamedTokensPolygon
+        : never;
+
+export interface DeploymentNamedTokensAssetsArbitrum {
```

Comment:
named tokens are mln, and weth only. Strange that typescript is not complaining

Reply:
Updated.